.forge-snapshots/PositionManager_multicall_initialize_mint.snap

```diff
@@ -1 +1 @@
-462111
\ No newline at end of file
+461714
```

Comment:
i wont claim i understand why its gone down 400 gas 😂

Reply:
i think because we're using safe transfer and that does token transfers in assembly